lib/outbrain/api/relation.rb

```diff
@@ -0,0 +1,16 @@
+module Outbrain
+  module Api
+    class Relation
+      include Enumerable
+      attr_accessor :relations, :totalDataCount, :overAllMetrics, :aggregatedBy 
```

Comment:
makes complete sense and is consistent with how we treat all the rest of the fields.